test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -471,10 +517,11 @@ def forward(self, x):
         self.run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
             Model(),
             (x,),
+            model_type=self.model_type,
             additional_test_inputs=[((x2,),)],
         )
 
-    @pytorch_test_common.xfail(
+    @pytorch_test_common.skip_model_type_is_not_exported_program_test(
```

Comment:
Is there a way for the new decorator to retain the finer grained failure conditions? E.g. xfail, skip, skip_for_ort_version_x, etc.

Reply:
For some reason there is only `skip` with filter options for failures; I've kept this behavior here, but indeed it would be nice to have `xfail_if_this_or_that` or a generic API which covered both.

Tracking through https://github.com/pytorch/pytorch/issues/113724